force-app/main/default/triggers/SameWheelValidation.trigger

```diff
@@ -0,0 +1,13 @@
+trigger SameWheelValidation on Wheel__c (before insert, before update) {
```

Comment:
Jezeli chodzi o samo formatowanie i wyglad kodu to przyjelo sie u nas ze soqle pisane sa w jednej linii (za wyjatkiem jakis mega dlugich, wtedy sa dzielone na kilka). Petle i warunki maja spacje przed i po nawiasach (for tez przed i po :) czyli na przyklad: 
`for (Wheel__c wheel : Trigger.New) {`